src/OSPSuite.Core/Commands/AddImporterConfigurationToProjectCommand.cs

```diff
@@ -19,6 +19,9 @@ public AddImporterConfigurationToProjectCommand(ImporterConfiguration configurat
 
       protected override void ExecuteWith(IOSPSuiteExecutionContext context)
       {
+         if (string.IsNullOrEmpty(_configuration.Id))
```

Comment:
Why do we trigger a command for an invalid configuration? The command will be added to the stack of executed command even though nothing happened. I think this is not ok

Reply:
You are right, I already cloned the issue on PKSim [1794](https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/1794) and fix it there, so the command will not be triggered on the first place. I would still keep this check to avoid crash on non-intended usage.

Reply:
hum...I almos twant to it to crash to be honest so that we are aware of non intended usage. THis SHOULD NEVER HAPPEN if we are calling this stuff properly so IMO, this is an exception that should be thrown

Reply:
Yes, you are right... I already remove the check.